dev_guide/builds.adoc

```diff
@@ -1557,3 +1557,114 @@ Docker and Source builds set the following labels on output images:
 |*io.openshift.build.source-location*
 |Source URL for the build
 |===
+
+
+[[using-external-artifacts]]
+= Using External Artifacts During The Build
+
+In order to use external artifacts during the build you don't need any special
+support from OpenShift platform. You could download and use any artifacts
+as you want.
```

Comment:
I'd start with a more objective sentence explaining what this section is about and why you would want to "Use external artifacts during a build".
A user reading the docs in a rush wants to tell if this is the right place to read and what he'll get out of continuing to read the next paragraphs.